lib/gadget/gadget-glue.c

```diff
@@ -585,4 +609,22 @@ frida_objc_api_try_get (void)
   return api;
 }
 
+static gboolean
+frida_dylib_range_try_get (const char * apple[], GumMemoryRange * range)
+{
+  const char * entry;
+  int i = 0;
```

Comment:
Nit: for consistency it would be good to use `gchar` and `gint` here, and perhaps use `guint` for the index while at it.